packages/@glimmer/runtime/lib/compiled/expressions/concat.ts

```diff
@@ -1,17 +1,15 @@
-import type { Dict, Maybe } from '@glimmer/interfaces';
 import type { Reference } from '@glimmer/reference';
 import { createComputeRef, valueForRef } from '@glimmer/reference';
-import { enumerate } from '@glimmer/util';
 
 export function createConcatRef(partsRefs: Reference[]) {
   return createComputeRef(() => {
-    let parts = new Array<string>();
+    const parts = [];
```

Comment:
The previous annotation was for TypeScript reason (so that it won't let you `push` arbitrary value into it later). I think we still need that right?

Reply:
good catch! Fixed